test/files/neg/compile-time-only-a.check

```diff
@@ -64,9 +64,6 @@ compile-time-only-a.scala:60: error: C7
 compile-time-only-a.scala:61: error: C7
   val c707a: List[C7] = ???
              ^
-compile-time-only-a.scala:62: error: C7
-  val c707b = List[C7]()
-                   ^
```

Comment:
could be worth updating the test, commenting why no error is issued here, and maybe add a variant that's not eliminated by magic.